components/form/form-element-mixin.js

```diff
@@ -210,6 +210,7 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 
 	validationCustomConnected(custom) {
 		this._validationCustoms.add(custom);
+		this.requestValidate(true);
```

Comment:
Inputs don't validate by default initially though, so this is a breaking change in that any input using a custom validator will now be validating immediately when the page loads vs. those without custom validators will not.

I think a change like this needs to be coupled with a new `validate-when-connected` attribute that when present would automatically call `requestValidate` during `firstUpdated` and also whenever a new custom validator is connected.